apps/dialtone-documentation/docs/.vuepress/client.js

```diff
@@ -33,6 +34,7 @@ export default defineClientConfig({
     app.component('UtilityClassTable', UtilityClassTable);
     app.component('ComponentClassTable', ComponentClassTable);
     app.component('TokenTable', TokenTable);
+    app.component('AllTokens', AllTokens);
```

Comment:
If you're using this component on a single page, you should import it there, not globally.